alacritty/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
 name = "alacritty"
 version = "0.4.2-dev"
-authors = ["Joe Wilm <joe@jwilm.com>"]
+authors = ["Christian Duerr <contact@christianduerr.com", "Joe Wilm <joe@jwilm.com>"]
```

Comment:
Bout time. ;)

Reply:
We can put this line in front `The alacritty contributors`, like winit is doing btw.